bringup/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+# ###############################################################################
+# Set minimum required version of cmake, project name and compile options
+# ###############################################################################
+cmake_minimum_required(VERSION 3.5)
+project(c3pzero_bringup)
+
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 17)
+endif()
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+# ###############################################################################
+# Find ament packages and libraries for ament and system dependencies
+# ###############################################################################
```

Comment:
`find_package` is another well-understood, popular CMake concept so there's little to be gained by quadrupling the line count with this comment.